### ISSUE TITLE
9.2.4.7/10.1 When the toggle buttons (see screenshot) receive keyboard focus, they do not meet the corresponding contrast requirements. Since the focus indication relies solely on a color change, the contrast ratio between the focused and unfocused states should be at least 3:1.

### ISSUE THREAD

**Issue by szaimen** [nextcloud/server#42264]
https://report.bitvtest.de/cbedb5ef-4815-4cba-a83d-995599c178c3.html#checkpoint-91f2646afa-v10-n1

9.2.4.7/10.1 When the toggle buttons (see screenshot) receive keyboard focus, they do not meet the corresponding contrast requirements. Since the focus indication relies solely on a color change, the contrast ratio between the focused and unfocused states should be at least 3:1.

under: settings/admin/security

![image](https://github.com/nextcloud/server/assets/42591237/1fb91e63-bcde-4b41-859a-9790a8188cdf)
